README.md

```diff
@@ -21,7 +21,14 @@ installation as a subdirectory of
 
 ## Restore packages
 
-	`mono src/.nuget/NuGet.exe restore src/Roslyn.sln`
+	mono src/.nuget/NuGet.exe restore src/Roslyn.sln
+
+On a fresh Mono install [that will fail with a certificate error](http://www.mono-project.com/docs/faq/security/).
```

Comment:
A fresh Mono 3.12 install from the Xamarin repos won't hit this since the certificate sync happens during package installations. It only affects installing Mono from source, so I'd suggest changing this message to reflect that.